docs/execution-providers/CUDA-ExecutionProvider.md

```diff
@@ -88,6 +88,12 @@ This flag is only supported from the V2 version of the provider options struct w
 
 Default value: 0
 
+### enable_cuda_graph
+Check [using CUDA Graphs in the CUDA EP](../performance/tune-performance.md#Using-CUDA-Graphs-in-the-CUDA-EP) for details on what this flag does.
```

Comment:
The LinkChecker can't find this anchor.

Reply:
That is strange. When I click on the link in the md file, I am routed to the right place.

Reply:
Fixed. I didn't realize that the LinkChecker doesn't like upper-casing.